source/components/Dropdown/index.js

```diff
@@ -104,42 +133,77 @@ Dropdown.propTypes = {
      * React Component to display as the Dropdown Content
      */
     children: PropTypes.node,
+
     /**
-     * Whether or not dropdown should have a slight drop shadow
+     * Should dropdown content be scrollable
      */
     contentScrollable: PropTypes.bool,
+
     /**
-     * Hides chevron in dropdown toggle
+     * Should dropdown content be left-aligned with the dropdown toggle
      */
     dropdownLeftAligned: PropTypes.bool,
+
     /**
-     * Whether or not dropdown should have a drop shadow (darker than the one produced by hasShadow)
+     * Should dropdown content be right-aligned with the dropdown toggle
      */
     dropdownRightAligned: PropTypes.bool,
+
     /**
-     * Is it a nested dropdown
+     * Whether or not dropdown should have a drop shadow (darker than the one produced by hasShadow)
      */
     hasDarkShadow: PropTypes.bool,
+
     /**
-     * Should dropdown content be left-aligned with the dropdown toggle
+     * Whether or not dropdown should have a slight drop shadow
      */
     hasShadow: PropTypes.bool,
+
     /**
      * is active
      */
     isActive: PropTypes.bool,
+
     /**
-     * Should dropdown content be right-aligned with the dropdown toggle
+     * Is it a nested dropdown
      */
     isLevel2: PropTypes.bool,
+
     /**
-     * Should dropdown content be scrollable
+     * if the top of nested dropdown content should be positioned at the same height as toggle
+     */
+    isStickedToParent: PropTypes.bool,
+
+    /**
+     * Hides chevron in dropdown toggle
      */
     noChevron: PropTypes.bool,
+
+    /**
+     * Removes span around element passed in the "toggle" prop
+     */
+    shouldNotWrapToggle: PropTypes.bool,
+
     /**
      * React Component to display as a dropdown toggle
      */
     toggle: PropTypes.node.isRequired,
+
+    /**
+     * HTML attributes to add to toggle
+     */
+    // eslint-disable-next-line react/forbid-prop-types
```

Comment:
WHat's the shape of the data we expect? Do nested object is ok? What about the one that has keys made from numerics? Consider using `shape` instead.
https://reactjs.org/docs/typechecking-with-proptypes.html